typings/objection/index.d.ts

```diff
@@ -774,6 +774,10 @@ declare namespace Objection {
     ): SingleQueryBuilder<QB>;
   }
 
+  interface GraphExpressionObjectMethod<QB extends AnyQueryBuilder> {
+    (): any;
```

Comment:
I really think we can improve the types here. I was playing with `ModelRelations` but couldn't quite get it right. Any suggestions for this? I think we can access the current Model via `QB`?

Reply:
We probably could, but I don't think it's worth the trouble at this point. I'd guess this method is rarely used.

Reply:
Sounds good, I will leave as-is.